explainer.md

```diff
@@ -55,28 +55,28 @@ Provide web apps with efficient access to built-in (software and hardware) media
 
 We build on top of [WHATWG Streams](https://streams.spec.whatwg.org/) (in particular [TransformStreams](https://streams.spec.whatwg.org/#ts-class)) and [MediaStreamTracks](https://www.w3.org/TR/mediacapture-streams/#dom-mediastreamtrack).
 
-**EncodedAudioPacket**s and **EncodedVideoFrame**s provide access to codec-specific encoded media bytes so they may be transported, saved, etc.
+**EncodedAudioChunk**s and **EncodedVideoChunk**s provide access to codec-specific encoded media bytes so they may be transported, saved, etc.
 
-**DecodedAudioPacket**s and **DecodedVideoFrame**s provide opaque handles for passing to/from other APIs (such as to/from MediaStreamTracks).   
+**AudioPacket**s and **VideoFrame**s may be passed opaque handles to other APIs (such as to/from MediaStreamTracks). Alternatively, AudioPacket may expose an [AudioBuffer](https://webaudio.github.io/web-audio-api/#audiobuffer) for rendering via [AudioWorklet](https://webaudio.github.io/web-audio-api/#audioworklet), while VideoFrame may expose an ImageBitmap for efficient rendering to canvas.
```

Comment:
"may be passed opaque handles": passed *as* opaque handles?

It's increasingly looking like they will not be opaque, maybe just stick with the second part.

Reply:
Done. Just kept the second part.

Reply:
Actually, going back to keeping it with the typo fixed (your first suggestion). Not that I disagree about  the overall direction, but we still have TrackWriters in the explainer and to some extent landed in Chrome... so seems sane to keep it for now.

Reply:
I re-ordered them to put canvas/audioworklet first.